app/views/shared/_cancel.html.slim

```diff
@@ -1,4 +1,6 @@
 - cancel = sign_up_or_idv_no_js_link || link
+- decorated_session = decorated_session
```

Comment:
@monfresh I had originally relied only on inheritance, but I was getting errors. Do you know if there's anything else that I need to do to make `decorated_session` available?